Example/AppDelegate+CarPlay.swift

```diff
@@ -452,7 +449,6 @@ extension GeocodedPlacemark {
 
 // MARK: - CPListTemplateDelegate methods
 
-@available(iOS 12.0, *)
```

Comment:
We also have `#if canImport(CarPlay)` statements in the codebase. I wonder if it can also be removed.

Reply:
Thanks for the suggestion, I removed these checks.

----

These checks were added in 2018. In the [2018 guide](https://developer.apple.com/carplay/documentation/CarPlay-Navigation-App-Programming-Guide.pdf) minimum requirements described as 
> CarPlay navigation apps require iOS 12 and Xcode 10

In fact most public classes did not even contain such a check, e.g. [CarPlayManager](https://github.com/mapbox/mapbox-navigation-ios/blob/main/Sources/MapboxNavigation/CarPlayManager.swift#L1), [NavigationGeocodedPlacemark](https://github.com/mapbox/mapbox-navigation-ios/blob/main/Sources/MapboxNavigation/NavigationGeocodedPlacemark.swift#L2)